akka-docs/src/main/paradox/classic/supervision.md

```diff
@@ -0,0 +1,256 @@
+# Classic Supervision
```

Comment:
this is a 1:1 copy of the old classic supervision as I thought that is different enough to keep the docs for.

Reply:
we have a pretty random structure for /classic/*.md vs *-classic.md

Reply:
Hmmm. I like putting new classic stuff in a folder best, but either way is fine by me.

Reply:
Discussion ended with use `-classic.md` for now. Possibly we'll change the file structure of the docs later.